cron.yaml

```diff
@@ -36,7 +36,7 @@ cron:
 
 - description: detect and flag tests with high flaky rates
   url: /api/file_flaky_issue_and_pr?threshold=0.02
-  schedule: every wednesday 9:00
+  schedule: every wednesday 9:00,9:30,10:00,10:30,11:00
```

Comment:
I saw your comment in the issue this is fixing; why not make this run in off peak hours? Does it need to be run during hours? What about every wednesday at 4:00 am?

Reply:
The time is UTC. Converting to PST these would be 1:00am - 3:00am.